src/index.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+const app=express();
+
+app.get('/',(req,res)=>{
+    res.send('hello?')
+});
+
+app.listen(3000,()=>{
```

Comment:
Instead of hard coding the port in ```app.listen``` function you should do it like this ```const port = process.env.PORT || 3000``` and use the port variable as a parameter. Basically, you never know which port will be allocated to your app when you deploy it.